### PR TITLE
[Doc-Improvement][Update the Initiate actions from messages description][2713754]

### PR DESCRIPTION
**Pull request by v-ajiths**
Updated the description

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cd65d81a78238cdecec00c1da9cd2ad44bef9ab4/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed7e68bd-1fe4-499d-84c4-bc463e589477) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/48759f0c22a376100bb12d4752102201ee017098/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e6855c67-dc95-4a90-be7b-6f3d4b9119d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/de769bcc8a7501e71ca933af0f35b287a5c63190/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a6a33ae7-255e-472a-a604-506bbb9a49f4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6ccb57e12700685405c5bbfb72ccbe4d6f6421f4/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54301444-0c6a-44f9-a6ca-84be221b5b04) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/messaging-extension-v3/create-extensions.md [hunk 1]
```diff
@@ -124,7 +124,9 @@ To initiate actions from a message extension, set the `type` parameter to `actio
 
 ### Initiate actions from messages
 
-You can initiate actions from the compose message area and also use your message extension to initiate an action from a message, which allows you to send the message contents to your bot for processing. Optionally, you can reply to that message with a response using the method described in [Responding to submit](#responding-to-submit). The response will be inserted as a reply to the message that your users can edit before submitting. Your users can access message extension from the overflow `...` menu and then selecting `Take action` as in the following image:
+You can initiate actions from the compose message area and from a message using your message extension, which allows you to send the message contents to your bot for processing. You can optionally respond to that message using the method described in [Responding to submit](#responding-to-submit). The response includes as a reply to the message, which users can edit before submitting.
```
Comment (surbhigupta12) [inline msteams-platform/resources/messaging-extension-v3/create-extensions.md]: @Wajeed-msft, please verify for technical accuracy

Comment (Wajeed-msft) [reply msteams-platform/resources/messaging-extension-v3/create-extensions.md]: Looks good to me.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2940b5a49beba3aeaf4679ca91ad44199ccb20ad/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f18adf3f-a9c0-4885-8e3b-cfe19f7ac581) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/abecf691da2ff45beb21ddc18e63d1949368ef50/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/25fef4bc-331e-4f1f-a96d-972b1d2a5366) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/60789401ac90476fcde8db62850148c0354bb00a/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8aa3cf64-7b40-41b4-94ca-6863e856a86b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/messaging-extensions/messaging-extensions-common.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ecb8931d885b56c43915c8a02477d940c9d5105e/msteams-platform%2Fincludes%2Fmessaging-extensions%2Fmessaging-extensions-common.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd055cfd-df7a-4b6c-be4d-da6b71dbfe58) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/v3-to-v4-pointer-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ecb8931d885b56c43915c8a02477d940c9d5105e/msteams-platform%2Fincludes%2Fv3-to-v4-pointer-me.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/111d6a1f-e018-44e7-adbb-1689a96812d2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ecb8931d885b56c43915c8a02477d940c9d5105e/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c852a0c-4b33-473f-8c69-dd2b2598d765) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/messaging-extensions/messaging-extensions-common.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f491318be1937d67fca2725aef000d7b54a7ace1/msteams-platform%2Fincludes%2Fmessaging-extensions%2Fmessaging-extensions-common.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0630723b-4f6f-445c-a7a0-5863f1ca9b1a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/v3-to-v4-pointer-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f491318be1937d67fca2725aef000d7b54a7ace1/msteams-platform%2Fincludes%2Fv3-to-v4-pointer-me.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fda11df3-7d43-452f-bed7-53f0734e48e8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f491318be1937d67fca2725aef000d7b54a7ace1/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba800719-9e69-4c71-acd8-ecaa2ca3dc60) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/messaging-extensions/messaging-extensions-common.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae273b0ebb812d95db5e898512c45b41a5fbd4df/msteams-platform%2Fincludes%2Fmessaging-extensions%2Fmessaging-extensions-common.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/78ea8ab3-899c-48c3-b50a-41a284b5abc0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/v3-to-v4-pointer-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae273b0ebb812d95db5e898512c45b41a5fbd4df/msteams-platform%2Fincludes%2Fv3-to-v4-pointer-me.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8f86ed8-72e0-4769-b4f3-97b28c08e0bf) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae273b0ebb812d95db5e898512c45b41a5fbd4df/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a52f8e6-8329-460c-83ec-d44ed1a1e050) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/messaging-extensions/messaging-extensions-common.md](https://github.com/MicrosoftDocs/msteams-docs/blob/493a89a6ef7e37ce1a52ba8aa95894ef80f1970d/msteams-platform%2Fincludes%2Fmessaging-extensions%2Fmessaging-extensions-common.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1664302-ef7d-45b0-aa06-753a759de3cd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/v3-to-v4-pointer-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/493a89a6ef7e37ce1a52ba8aa95894ef80f1970d/msteams-platform%2Fincludes%2Fv3-to-v4-pointer-me.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f3974b60-476b-4d40-80c2-aa7c3b8f2a62) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/493a89a6ef7e37ce1a52ba8aa95894ef80f1970d/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38a3eac1-6c39-4598-8471-6a3808bdb499) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/messaging-extensions/messaging-extensions-common.md](https://github.com/MicrosoftDocs/msteams-docs/blob/65f2e4aded3f56ddb941d1915bc2028ae8a942d9/msteams-platform%2Fincludes%2Fmessaging-extensions%2Fmessaging-extensions-common.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6df928cf-fc12-4746-83d0-75fba7a1fc17) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/v3-to-v4-pointer-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/65f2e4aded3f56ddb941d1915bc2028ae8a942d9/msteams-platform%2Fincludes%2Fv3-to-v4-pointer-me.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4c65f28-8ec6-42b9-a0e8-44847e698f6c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/65f2e4aded3f56ddb941d1915bc2028ae8a942d9/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2fc3626c-0fbf-4c7b-8505-6cbb029f68d1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/messaging-extensions/messaging-extensions-common.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1b16db9085e8b0b42ebcd6ef45b34d3839ee372d/msteams-platform%2Fincludes%2Fmessaging-extensions%2Fmessaging-extensions-common.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1af3a134-041d-49c3-840d-6d2a079f3a04) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/includes/v3-to-v4-pointer-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1b16db9085e8b0b42ebcd6ef45b34d3839ee372d/msteams-platform%2Fincludes%2Fv3-to-v4-pointer-me.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/82ddd391-4bb5-4613-b14b-ad77b4fe6c1c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/messaging-extension-v3/create-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1b16db9085e8b0b42ebcd6ef45b34d3839ee372d/msteams-platform%2Fresources%2Fmessaging-extension-v3%2Fcreate-extensions.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5001b9b9-e146-441b-86ec-98a76822a0e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]

**Pull request merged**